pyseir/models/suppression_policies.py

```diff
@@ -22,6 +22,75 @@
 }
 
 
+def get_r0_from_rt(fips):
+    """
+    Load R0 as the maximum of Rt. if the fips is not found, the state level is
+    used.
+
+    Parameters
+    ----------
+    fips: str
+        State or County FIPS code.
+
+    Returns
+    -------
+    r0: float
+        Maximum (unmitigated) Rt value.
+    """
+    try:
+        df_rt = fit_results.load_Rt_result(fips)
+    except ValueError:
+        df_rt = fit_results.load_Rt_result(fips[:2])
+
+    df_rt = df_rt[df_rt['Rt_MAP_composite'].notnull()]
+    return df_rt['Rt_MAP_composite'].max()
+
+
+def generate_rt_inference_policy(t0, fips, future_suppression, R0=None):
+    """
+    Generate a suprpession policy based on the inferred Rt.
+
+    Parameters
+    ----------
+
+    t0: datetime
+        Initial start time.
+    fips: str
+        State or county FIPS
+    future_suppression: float or str
+        Future suppression level to use. If 'current', use the last value.
+    R0: float or NoneType
+        Natural unmitigated reproduction number. If None, this will lookup the
+        maximal observed R for the fips.
+
+    Returns
+    -------
+    suppression_model: callable
+        suppression_model(t) returns the current suppression model at time t.
+    """
+    if R0 is None:
+        R0 = get_r0_from_rt(fips)
+    try:
+        df_rt = fit_results.load_Rt_result(fips)
+    except ValueError:
+        df_rt = fit_results.load_Rt_result(fips[:2])
+
+    df_rt = df_rt[df_rt['Rt_MAP_composite'].notnull()]
+
+    days = [(t - t0).days for t in df_rt['Rt_MAP_composite'].index.to_pydatetime()]
```

Comment:
I need to think more about whether this shift is right...  

Rt is known at a given time, but we may want to allow for a small shift to account for delays in that vs the simulation.  Right now this is shifting the entire Rt timeseries along with the simulation, which may not be what we want (i.e. we may want to allow it to move *within* the simulation).